src/Block.cpp

```diff
@@ -238,6 +238,7 @@ Block::~Block(void)
 	deleted_stms.clear();
 
 	local_vars.clear();
+	local_var_set.clear();
```

Comment:
I agree. There was some reason that I didn't rip out the old collections completely; I think it was simply a *lot* of work in a lot of places. In these patches I only did the parts that seemed to make a difference in performance according to my profiling.
Thanks for the review, I will see when I get around to reworking the issues you mentioned.